pyproject.toml

```diff
@@ -51,6 +51,7 @@ doc = [
     "numpydoc==1.8.0",
     "pandas==2.2.3",
     "Pillow>=9.0",
+    "plotly==5.24.1", # plotly 6.0 is blocked because of https://github.com/plotly/plotly.py/issues/5012
```

Comment:
Plotly 6.0.0 Issue: We cannot use Plotly 6.0.0 as there is a breaking issue on their side. I have added the issue link in the pyproject.toml file next to the dependency.( i feel that adding anywidget as a dependency might fix this issue from their migration guide) i will explore this

Reply:
Let's try with plotly==6.0.1. It looks like this issue was just fixed.